services/common/src/components/reports/ReportDetailsForm.tsx

```diff
@@ -218,80 +241,125 @@ const ReportDetailsForm: FC<ReportDetailsFormProps> = ({
-            </b>
-          }
-          type="warning"
-          showIcon
-          style={{ marginBottom: "32px" }}
-        />
+      {(isEditMode || !formValues.mine_report_guid) && system !== SystemFlagEnum.core && (
+        <>
+          {isPRR && (
+            <Alert
+              message=""
+              description={
+                <b>
+                  You are submitting a permit required report. If you intended to submit a code
+                  required report, please go back and select it on the report type screen.
+                </b>
+              }
+              type="warning"
+              showIcon
+            />
+          )}
+          <Alert
+            message=""
+            description={
+              <b>
+                If your report package relates to more than one Code requirement, please submit as
+                separate report submissions.
+              </b>
+            }
+            type="info"
+            showIcon
+            style={{ marginBottom: "32px" }}
+          />
+        </>
       )}
-      <Typography.Title level={3} id="report-type">
-        Report Type
-      </Typography.Title>
 
       <FormWrapper
         name={FORM.VIEW_EDIT_REPORT}
         onSubmit={handleSubmit}
         isEditMode={isEditMode}
         initialValues={initialValues}
       >
+        {system === SystemFlagEnum.core && formButtons}
+        <Typography.Title level={3} id="report-type">
+          Report Type
+        </Typography.Title>
         <Row gutter={[16, 8]}>
+          {system === SystemFlagEnum.ms && (
+            <>
+              <Col md={12} sm={24}>
+                <BaseViewInput
+                  label="Report Type"
+                  value={report_type && MINE_REPORTS_ENUM[report_type]}
+                />
+              </Col>
+              {isPRR && (
+                <Col md={12} sm={24}>
+                  <BaseViewInput label="Permit Number" value={permit?.permit_no} />
+                </Col>
+              )}
+              {selectedPermitCategory && (
+                <Col md={12} sm={24}>
+                  <BaseViewInput
+                    label="Permit Condition Category"
+                    value={selectedPermitCategory.label}
+                  />
+                </Col>
+              )}
+            </>
+          )}
           {system === SystemFlagEnum.core && (
-            <Col span={24}>
+            <>
+              <Col span={24}>
+                <Field
+                  name="report_type"
+                  id="report_type"
+                  required
+                  disabled={true}
+                  props={{
+                    isVertical: true,
+                  }}
+                  label="What is the type of the report?"
+                  component={RenderRadioButtons}
+                  validate={[requiredRadioButton]}
+                  customOptions={[
+                    { label: MINE_REPORTS_ENUM.CRR, value: REPORT_TYPE_CODES.CRR },
```

Comment:
I should have made the string values in the options be from an enum, thanks for making the change here.